packages/camera/camera_avfoundation/example/lib/camera_controller.dart

```diff
@@ -502,7 +510,7 @@ class Optional<T> extends IterableBase<T> {
   /// Gets the Optional value.
   ///
   /// Throws [StateError] if [value] is null.
-  T get value {
+  T? get value {
```

Comment:
Is this and the changes below new? Why would a method that thows if the value is null, and can thus never return null, need to be made nullable?

Reply:
this was result of some previous merge. restored to T.